src/app/fdctl/run/tiles/dedup.seccomppolicy

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
I like the policy definiton file format, except for this weird newline for empty variables. Not sure if there's a cleaner way to do this but it would be nice.

Reply:
100% agree with you. I'll figure something out. Thanks for raising!

Reply:
This has been adressed.